### DISCUSSION TITLE
I am new to github

### DISCUSSION THREAD

**Discussion by jgax87** [js-cookie/js-cookie#689] [Q&A] [answered]
Hello, i am new to this. I found js-cookie through stackoverflow...

anyway i want to use this one my website, and i downloaded the zip but I can't find js.cookie.js or js.cookie.mjs. searched through all the folders and i can't find it. 

how do i go about using this or how to embedded within my website? It's already uploaded to my server.

**Comment by tobiasroeder** [comment, marked as answer]
This might help: https://www.jsdelivr.com/package/npm/js-cookie?path=src
